frontend/src/components/TaskDetailsBody/TaskDetailsBody.tsx

```diff
@@ -8,18 +8,28 @@ import {
   RadioButtonUnchecked,
   Flag,
 } from '@mui/icons-material/';
-import { useCompletedStore } from '../../store/task.store';
-
+import api from '../../api';
+import { useCompletedStore, useTaskStore, useProjectStore } from '../../store/task.store';
 import { Task } from '../../types/Task';
+import { format } from 'date-fns';
+import { Dropdown } from '../dropdown/CustomDropDown';
 
 type TaskDetailsBodyType = {
   task: Task;
 };
 
 const TaskDetailsBody = ({ task }: TaskDetailsBodyType) => {
   const { completed, setCompleted } = useCompletedStore();
+  const { setTask } = useTaskStore();
   const completedStatus = completed[`${task.id}`] ?? false;
+  const [title, setTitle] = useState(task.title);
   const [description, setDescription] = useState(task.description);
+  const defaultDueDate = task.due_date ? format(new Date(task.due_date), "yyyy-MM-dd'T'HH:mm") : '';
+  const [dueDate, setDueDate] = useState(defaultDueDate);
+  const { projectTitle } = useProjectStore();
+  const priorityArray = ['easy', 'medium', 'hard'];
+  const usersArray = ['Arash', 'Mike']; // This array will be replaced by users data from database
```

Comment:
Why have it here then? Why not just have empty arrays?

Reply:
because the end poin still not ready from backend